python_modules/dagster/dagster/_core/definitions/resource_invocation.py

```diff
@@ -84,14 +84,14 @@ def _check_invocation_requirements(
 
     # Construct a context if None was provided. This will initialize an ephemeral instance, and
     # console log manager.
-    init_context = init_context or build_init_resource_context()
+    _init_context = init_context or build_init_resource_context()
```

Comment:
not your problem, but i wish we had a suitable naming convention for this type variable where you reassign it with a default valiue.